test/check-application

```diff
@@ -73,6 +73,24 @@ class TestNavigator(testlib.MachineCase):
         b.click(f"#create-link-{filetype}")
         b.click("button.pf-m-primary")
 
+    # TODO: use https://github.com/cockpit-project/cockpit/pull/19890 when landed
+    def allowDownload(self):
+        b = self.browser
+        if b.cdp.browser.name == "chromium":
+            b.cdp.invoke("Page.setDownloadBehavior", behavior="allow", downloadPath=b.cdp.download_dir)
+
+    def waitDownloadFile(self, filename, expected_size=None, content=None):
+        b = self.browser
+        filepath = os.path.join(b.cdp.download_dir, filename)
+
+        testlib.wait(lambda: os.path.exists(filepath))
+        if expected_size is not None:
+            self.assertEqual(os.stat(filepath).st_size, expected_size)
+
+        if content is not None:
+            with open(filepath) as fp:
+                self.assertEqual(fp.read().strip(), content)
```

Comment:
This shouldn't strip() here, that's weird -- `content` should be exactly as expected